ext/random/tests/03_randomizer/methods/nextFloat_spacing.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+Random: Randomizer: nextFloat(): Return values are evenly spaced.
```

Comment:
Not sure if this is actually a good test, it relies quite a bit on the knowledge about the internal implementation.

I'd likely drop it. Opinions?